test-definitions/fern/apis/failing-audiences/definition/api.yml

```diff
@@ -0,0 +1,16 @@
+name: audiences
```

Comment:
It isn't easy to understand what edge case you are trying to test with `failing-audiences`. could there be a better name for this definition?

Reply:
This is just a temporary fixture, until the bugs get solved, and then it will be deleted!